docs/guide/persistence_cache.md

```diff
@@ -120,7 +120,8 @@ parameters:
 Redis is used via [Redis pecl extension](https://pecl.php.net/package/redis).
 
 See [Redis Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html#configure-the-connection)
-for information on how to configure Redis.
+for information on how to configure Redis. Make sure to configure redis with the right eviction policy. So, you can be safe that tag <=> cache
```

Comment:
```suggestion
for information on how to configure Redis. Make sure to configure Redis with the right eviction policy. 
```

Reply:
> See [Redis Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html#configure-the-connection)

Actually we don't ship using RedisAdapter anymore since 2.5, so while the Symfony doc link is correct on all options and DSN, the adapter name is different so we should also explain that here.


So maybe:

```suggestion
for information on how to connect to Redis. Main difference in eZ Platform is that we ship with a optimized Adapter called RedisTagAwareAdapter _(see `app/config/cache_pool/cache.redis.yml` for configuration example)_.

This optimized adapter depends on configuring a eviction policy which does *not* risk deleting Tag relation data when reaching memory limits.
```